cli_tools/common/utils/daisy/daisy_utils.go

```diff
@@ -84,28 +85,46 @@ func UpdateAllInstanceNoExternalIP(workflow *daisy.Workflow, noExternalIP bool)
 	if !noExternalIP {
 		return
 	}
-	updateAllInstanceAccessConfig(workflow, func() []*compute.AccessConfig {
-		return []*compute.AccessConfig{}
-	})
-}
-
-func updateAllInstanceAccessConfig(workflow *daisy.Workflow, accessConfigProvider func() []*compute.AccessConfig) {
-	for _, step := range workflow.Steps {
-		if step.IncludeWorkflow != nil {
-			//recurse into included workflow
-			updateAllInstanceAccessConfig(step.IncludeWorkflow.Workflow, accessConfigProvider)
-		}
+	daisy.WorkflowStepIterator(workflow, func(step *daisy.Step) {
```

Comment:
IterateWorkflowSteps is better from my perspective. Iterator sounds more like a object than a operation.

Reply:
renamed and made a member func